include/seqan/seq_io/sequence_file.h

```diff
@@ -387,30 +532,22 @@ writeRecord(FormattedFile<Fastq, Output, TSpec> & file,
  * @signature void writeRecords(fileOut, metas, seqs, quals);
  * @see SeqFileOut#writeRecord
  */
-
-template <typename TSpec, typename TIdStringSet, typename TSeqStringSet>
-inline void
-writeRecords(FormattedFile<Fastq, Output, TSpec> & file,
-             TIdStringSet const & meta,
-             TSeqStringSet const & seq)
+template <typename TFile, typename TIdStringSet, typename TSeqStringSet>
+inline void writeRecords(TFile & file, TIdStringSet const & meta, TSeqStringSet const & seq)
```

Comment:
Hm, I though that the whole point of writeRecord**s** is that tag dispatching happens once, but here it looks like tag dispatching happens for each record?